Lib/ufo2ft/featureWriters/ast.py

```diff
@@ -85,8 +85,12 @@ def iterClassDefinitions(feaFile, featureTag=None):
 }
 
 
-def makeLookupFlag(name=None, markAttachment=None, markFilteringSet=None):
-    value = 0 if name is None else LOOKUP_FLAGS[name]
+def makeLookupFlag(name=None, flags=None, markAttachment=None, markFilteringSet=None):
+    value = 0
+    if name:
+        value = LOOKUP_FLAGS[name]
+    if flags:
```

Comment:
maybe instead of adding a new `flags` parameter, rename the initial `name` parameter to `flags` and accept either a single string or a sequence of strings? This is mostly an internal helper function, and the first argument is only used as positional parameter. I don't like that we have two params `name` and `flags` which basically do the same thing